monkestation/code/modules/bloodsuckers/structures/bloodsucker_crypt.dm

```diff
@@ -289,10 +294,6 @@
 		return FALSE
 	var/disloyalty_requires = RequireDisloyalty(user, target)
 
-	if(HAS_TRAIT(target, TRAIT_MINDSHIELD))
```

Comment:
given the fact `/obj/structure/bloodsucker/vassalrack/proc/remove_loyalties(mob/living/target)` is a thing, I assume this check is a mistake, and that mindshielded victims should just fall under `VASSALIZATION_DISLOYAL` instead

Reply:
bloodsucker code is a pile of jank

Reply:
the worst part is the pascal cased proc names